openedx/features/philu_courseware/tests/test_competency_assessment_record_manager.py

```diff
@@ -0,0 +1,46 @@
+from django.test import TestCase
+from opaque_keys.edx.keys import UsageKey
+
+from openedx.features.philu_courseware.models import CompetencyAssessmentRecord
+from student.tests.factories import UserFactory
+
+from .factories import CompetencyAssessmentRecordFactory
+
+
+class CompetencyAssessmentRecordManagerTestCase(TestCase):
+
+    def setup_attempts(self, assessment_type='post'):
+        for question_number in range(5):
+            CompetencyAssessmentRecordFactory(
+                user=self.user,
+                assessment_type=assessment_type,
+                question_number=question_number + 1
+            )
+
+    def delete_post_assessment_attempts(self):
+        return CompetencyAssessmentRecord.objects.revert_user_post_assessment_attempts(
+            user=self.user, problem_id=self.problem_id
+        )
+
+    def setUp(self):
+        self.user = UserFactory()
+        self.problem_id = UsageKey.from_string(
+            'block-v1:PUCIT+IT1+1+type@problem+block@7f1593ef300e4f569e26356b65d3b76b'
+        )
```

Comment:
Since problem_id is a static value, either we can move this to class inside `delete_post_assessment_attempts`. `setUp` will be called for each test, so there is no need to initialize a static value separately for each test